fdbbackup/backup.actor.cpp

```diff
@@ -954,10 +954,10 @@ static void printAgentUsage(bool devhelp) {
 	       "  --logdir PATH  Specifes the output directory for trace files. If\n"
 	       "                 unspecified, defaults to the current directory. Has\n"
 	       "                 no effect unless --log is specified.\n");
-	printf("  --loggroup LOG_GROUP\n"
+	printf("  --loggroup LOG-GROUP\n"
```

Comment:
I would leave `LOG_GROUP` alone here, since it's not part of the argument name.

Reply:
Got it. Thanks!